core/loadpoint_api.go

```diff
@@ -205,42 +125,114 @@ func (lp *Loadpoint) SetPhases(phases int) error {
 	return nil
 }
 
-// GetTargetTime returns the target time
-func (lp *Loadpoint) GetTargetTime() time.Time {
+// GetLimitSoc returns the session limit soc
+func (lp *Loadpoint) GetLimitSoc() int {
+	lp.RLock()
+	defer lp.RUnlock()
+	return lp.limitSoc
+}
+
+// setLimitSoc sets the session limit soc (no mutex)
+func (lp *Loadpoint) setLimitSoc(soc int) {
+	lp.limitSoc = soc
+	lp.settings.SetInt(keys.LimitSoc, int64(soc))
+	lp.publish(keys.LimitSoc, soc)
+	// TODO locking and more values
+	// lp.publish(keys.EffectiveLimitSoc, lp.effectiveLimitSoc())
```

Comment:
@andig kannst du dich um dieses Todo kümmern? Ohne das klappt der Roundtrip in der UI nicht.

Reply:
Done. Falls das irgendwie ein Performancproblem wird hab ich auch schon eine Idee.